src/main/kotlin/frontend/components/AtlantaFxUtils.kt

```diff
@@ -1,16 +1,35 @@
 package frontend.components
 
 import atlantafx.base.controls.Card
+import atlantafx.base.controls.Notification
 import atlantafx.base.controls.ToggleSwitch
 import atlantafx.base.layout.InputGroup
+import atlantafx.base.theme.Styles
+import javafx.event.EventHandler
 import javafx.event.EventTarget
 import javafx.scene.Node
+import javafx.scene.layout.Region
+import javafx.scene.layout.StackPane
 import org.kordamp.ikonli.Ikon
 import org.kordamp.ikonli.javafx.FontIcon
+import org.kordamp.ikonli.material2.Material2OutlinedAL
 import tornadofx.*
 
 fun EventTarget.inputGroup(vararg children: Node, op: InputGroup.() -> Unit = {}) = opcr(this, InputGroup(*children), op)
 fun EventTarget.toggleSwitch(text: String? = null, op: ToggleSwitch.() -> Unit = {}) =
   opcr(this, ToggleSwitch(text), op)
 fun EventTarget.fontIcon(icon: Ikon, op: FontIcon.() -> Unit = {}) = opcr(this, FontIcon(icon), op)
 fun EventTarget.card(op: Card.() -> Unit = {}) = opcr(this, Card(), op)
+
+fun notify(element: StackPane, message: String) {
```

Comment:
create extension function, dude

Reply:
https://kotlinlang.org/docs/extensions.html#extensions-are-resolved-statically

Reply:
what should I extend?

Reply:
`EventTarget`